gunicorn/glogging.py

```diff
@@ -199,7 +204,11 @@ def setup(self, cfg):
                 self.access_log, cfg, self.syslog_fmt, "access"
             )
 
-        if cfg.logconfig:
+        if dictConfig and cfg.logconfig_dict:
```

Comment:
No problem :) I think making dictConfig an error on 2.6 and below would probably be best, but I'll wait until you're done with the review before changing anything.